src/scene/Entity.cpp

```diff
@@ -77,3 +84,60 @@ std::optional<Mat3x4f> Entity::getPreviousFrameLocalToWorldTransform() const
 
 	return formerTransformInfo.matrix;
 }
+
+void Entity::applyExternalAnimation(const Vec3f* vertices, std::size_t vertexCount)
+{
+	if (!std::holds_alternative<ExternalAnimator>(animator)) {
+		animator = ExternalAnimator(mesh->dVertices);
+	}
+	if (auto* externalAnimator = std::get_if<ExternalAnimator>(&animator)) {
```

Comment:
> I expect this to always be true, because if animator does not hold ExternalAnimator objecty, then such object is assigned to animator (line 91 above).

True. If-statement is not needed here. We can simply `std::get`